simglucose/simulation/env.py

```diff
@@ -34,7 +36,58 @@ def risk_diff(BG_last_hour):
 
 
 class T1DSimEnv(object):
+    '''
+    Observation:
+        Type: Box(4)
+        Num     Observation                 Min         Max
+        0       BG                          69.         351.
+        1       Insulin                     0.0         Inf
```

Comment:
Insulin is not in Observation. Since insulin is given by agent, it is already known by the agent, there is no need to put it in the observation.